pkg/backup/item_backupper.go

```diff
@@ -123,6 +123,11 @@ func (ib *defaultItemBackupper) backupItem(logger logrus.FieldLogger, obj runtim
 		log = log.WithField("namespace", namespace)
 	}
 
+	if exclude := metadata.GetLabels()["velero.io/exclude-from-backup"]; exclude == "true" {
```

Comment:
don't really need to declare `exclude` here, can just say `if metadata.GetLabels()["velero.io/exclude-from-backup"] == "true" {`